scout/server/blueprints/cases/templates/cases/case_report.html

```diff
@@ -1011,25 +1012,8 @@ <h4>Scout case analysis report</h4> - created on:&nbsp;<strong>{{report_created_
         </td>
         <td style="width:3%"></td>
         <td>
-          <table id="panel-table" class="table table-sm" style="background-color: transparent">
-            <thead>
-              <th colspan=2>Pop. frequency</th>
-            </thead>
-            <tbody>
-              {% for freq_name, value, link in variant.frequencies %}
-                <tr>
-                  <td>{{ freq_name }}</td>
-                  <td>
-                    {% if value %}
-                      <span class="badge bg-secondary">{{ value|human_decimal }}</span>
-                    {% else %}
-                      <span class="text-muted">Not annotated</span>
-                    {% endif %}
-                  </td>
-                </tr>
-              {% endfor %}
-            </tbody>
-          </table>
+          {{ frequencies_table(variant) }}
```

Comment:
Apply the same also to SVs:

<img width="1125" alt="image" src="https://github.com/user-attachments/assets/45108818-8742-40fe-aae1-a6ba817c7fbc" />